docker_etl/create_new_job.py

```diff
@@ -0,0 +1,94 @@
+import os
+import shutil
+from typing import Dict
+
+import click
+import jinja2
+
+from docker_etl.ci_config import update_config
+from docker_etl.file_utils import (
+    CI_JOB_NAME,
+    CI_JOB_TEMPLATE_NAME,
+    CI_WORKFLOW_NAME,
+    CI_WORKFLOW_TEMPLATE_NAME,
+    JOBS_DIR,
+    TEMPLATES_DIR,
+)
+
+
+def get_templates() -> Dict[str, str]:
+    """Get mappings of template name to template path."""
+    return {
+        name: os.path.join(TEMPLATES_DIR, name)
+        for name in os.listdir(TEMPLATES_DIR)
+        if os.path.isdir(os.path.join(TEMPLATES_DIR, name))
+    }
```

Comment:
I'd recommend `pathlib` over `os` for path manipulation in most place, but no need to rewrite anything.

```python
# type(TEMPLATES_DIR) == pathlib.Path
return {p.name: p for p in TEMPLATES_DIR.glob("*") if p.is_dir()}
# str(p) or p.as_posix() for a string to feed jinja
```

Reply:
Redid everything with pathlib and it's much cleaner